data/pools/transactionPool.go

```diff
@@ -681,7 +585,6 @@ func (pool *TransactionPool) addToPendingBlockEvaluatorOnce(txgroup []transactio
 func (pool *TransactionPool) addToPendingBlockEvaluator(txgroup []transactions.SignedTxn, recomputing bool, stats *telemetryspec.AssembleBlockMetrics) error {
 	err := pool.addToPendingBlockEvaluatorOnce(txgroup, recomputing, stats)
 	if err == ledgercore.ErrNoSpace {
-		pool.numPendingWholeBlocks++
```

Comment:
Removing this in the name of taking out old-style congestion code may be a mistake. It means we can't remove a transaction that has been in the pool too long from here, because we're not keeping track of a sort of "virtual round".

But, who cares? It'll get kicked out by `eval` before it gets into an actual block.